docs/usage.md

```diff
@@ -4,13 +4,134 @@
 
 > _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
 
-## Introduction
+## Input specifications
 
-### Running the pipeline
+The input data can be passed to nf-core/ampliseq in three possible ways using the `--input` parameter, either a folder containing zipped FastQ files, a tab-seperated samplesheet, or a fasta file to be taxonomically classified.
+
+Optionally, a metadata sheet can be specified for downstream analysis.
+
+### Direct FASTQ input
+
+The easiest way is to specify directly the path to the folder that contains your input FASTQ files. For example:
+
+```console
+--input 'path/to/data'
```

Comment:
```suggestion
--input 'path/to/data/'
```

Reply:
To specify that this is a folder as input? Unless this breaks the picking up of the contained FASTQs files

Reply:
That should not break anything, so thats fine.